### PR TITLE
Move AKS related instructions to IoT Packages project

### PR DESCRIPTION
**Pull request by sophokles73**

### CODE REVIEW

**Comment by calohmn** [comment]
I guess the commit [0b03961](https://github.com/eclipse/hono/pull/3354/commits/0b039615adf1777150629ef1f8215da5909baa24) doesn't belong in here.

File: site/documentation/content/deployment/create-kubernetes-cluster.md [hunk 1]
```diff
@@ -3,150 +3,6 @@ title = "Setting up a Kubernetes Cluster"
...
-
-<!--more-->
-
-Hono can be deployed to any Kubernetes cluster running version 1.11 or newer. This includes [OpenShift (Origin)](https://www.okd.io/) which is built on top of Kubernetes.
-
-The [Kubernetes web site](https://kubernetes.io/docs/setup/) provides instructions for setting up a (local) cluster on bare metal and/or virtual infrastructure from scratch or for provisioning a managed Kubernetes cluster from one of the popular cloud providers.
-
-<a name="Local Development"></a>
-
-## Setting up a local Development Environment
```
Comment (calohmn) [inline site/documentation/content/deployment/create-kubernetes-cluster.md:16]: There are still links to this chapter in the Getting Started guide (relref `/deployment/create-kubernetes-cluster#setting-up-a-local-development-environment`).

**Comment by sophokles73** [comment]
@calohmn would you mind taking another look?

**Review by calohmn — APPROVED** [review]
LGTM

**Pull request merged**